api/swagger.yaml

```diff
@@ -8263,3 +8263,62 @@ paths:
           format: "int64"
           required: true
       tags: ["Secret"]
+  /distribution/{name}/json:
```

Comment:
I would advise caution in placing the name directly in the path. While we do this for the registry, ensuring that all cases are captured is non-trivial, unless it is properly escaped.